pom.xml

```diff
@@ -386,7 +386,7 @@
             <dependency>
                 <groupId>com.google.protobuf</groupId>
                 <artifactId>protobuf-java</artifactId>
-                <version>3.21.9</version>
```

Comment:
There is another protobuf dependency below, `protobuf-java-util` with an unchanged version. Should both versions be same?

Reply:
sure, I will open a new pr instead of dependabots